app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -35,6 +35,48 @@ angular.module('dashboard')
             }
         }
 
+        function meetingStatusChanged(event) {
+            $log.debug("meetingStatusCtrl: meetingStatusChanged");
+            if (angular.isObject(event) && angular.isObject(self.meeting)) {
+                self.meeting.meetingStatus = event.MeetingStateType;
+            }
+        }
+
+        function topicStatusChanged(event) {
+            $log.debug("meetingStatusCtrl: topicStatusChanged");
+            if (angular.isObject(event) && angular.isObject(self.meeting) && angular.isArray(self.meeting.topicList)) {
+                for (var i = 0; i < self.meeting.topicList.length; i++) {
+                    var topic = self.meeting.topicList[i];
+                    if (angular.isObject(topic) && angular.equals(topic.topicGuid, event.TopicID)) {
+                        topic.topicStatus = event.TopicStateType;
+                    }
+                }
+            }
+        }
+
+        function proposalsStatusChanged(events) {
+            $log.debug("meetingStatusCtrl: proposalsStatusChanged");
+            if (angular.isArray(events) && angular.isObject(self.meeting) && angular.isArray(self.meeting.topicList)) {
+
+                var changedTopicGuidArray = [];
+                angular.forEach(events, function (topic) {
+                    if (angular.isObject(topic) && angular.isObject(topic.Proposal) && topic.Proposal.topicGuid) {
+                        changedTopicGuidArray.push(topic.Proposal.topicGuid);
+                    }
+                }, changedTopicGuidArray);
+
+                for (var j = 0; j < self.meeting.topicList.length; j++) {
+                    var topic = self.meeting.topicList[j];
+                    if (angular.isObject(topic)) {
+                        topic.isModified = (changedTopicGuidArray.indexOf(topic.topicGuid) > CONST.NOTFOUND);
+                        if (!isMobile && angular.equals(topic.topicGuid, selectedTopicGuid)) {
+                            StorageSrv.setKey(CONST.KEY.TOPIC, angular.copy(topic));
```

Comment:
meeting.html uses topic to show isModified state
<span ng-show="ctrl.topic.isModified">